### DISCUSSION TITLE
No-B̴̟͎̻͌̓́z̴̻͚̈́͠͝z̸̙͙͕̈́̿͆T̴̝̘̞̽͊͝ -thing loadsB̴̟͎̻͌̓́z̴̻͚̈́͠͝z̸̙͙͕̈́̿͆T̴̝̘̞̽͊͝

### DISCUSSION THREAD

**Discussion by lukeschwinn001** [Prodigy-Hacking/ProdigyMathGameHacking#836] [(B) Bug Discussion]
![2021-03-30 (2)](https://user-images.githubusercontent.com/81653126/113058036-cb22ea00-917b-11eb-807a-5890db6396ad.png)

**Comment by ghost** [comment]
Shouldn't you write this is issues?

**Reply by lukeschwinn001** [reply]
Oh yesss, sorry

**Comment by LeoBadeaux** [comment]
You guys think you're so cool using a "glitch text" generator. No. No. Not at all.

**Reply by ghost** [reply]
ikr
